data/Makefile

```diff
@@ -1,5 +1,5 @@
 # Size of a k-mer.
-k=32
+k=40
```

Comment:
Is this the k we are using for all human? Just wondering why we changed the default

Reply:
Yes we are using k40 for all human and higher k is more accurate.